network/p2p/tracer/gossipSubScoreTracer.go

```diff
@@ -252,10 +252,10 @@ func (g *GossipSubScoreTracer) logPeerScore(peerID peer.ID) bool {
 	}
 
 	if snapshot.IsWarning() {
-		lg.Warn().Msg(PeerScoreLogMessage)
+		lg.Debug().Msg(PeerScoreLogMessage)
```

Comment:
from the function name `IsWarning()`, it seems like this should remain a warning. Is it over reporting?

Reply:
I do saw lots of warning log here, which I have mentioned to @yhassanzadeh13 
![image](https://github.com/onflow/flow-go/assets/811374/94675be8-3c6b-40d4-84bd-9035c49d9b21)

Reply:
It should be a `Warn` level, the reason it appears frequently is the IP collocation factor, which we need to fix it.